### DISCUSSION TITLE
Just wondering if this is the correct Nexus before making a purchase

### DISCUSSION THREAD

**Discussion by SpookyDatum** [seemoo-lab/nexmon_csi#217] [Q&A]
I am currently going to school for Cyber Security and am trying to create a few toys to get hands on experience.  Just want to make sure I am purchasing the correct phone.  It is a bit expensive, and if I dont get this right I am fairly certain my wife will kill my daughters father xD

Is this correct?
https://www.amazon.com/LG-Nexus-5X-Unlocked-Smartphone/dp/B0178GEA04
and...
https://www.amazon.com/Motorola-XT1103-Unlocked-Cellphone-Midnight/dp/B00UXUGNIK

Any help is appreciated!

**Comment by matthiasseemoo** [comment]
Get one of the phone that we have in our list. The Nexus 5X is different from the Nexus 5 and does not work with Nexmon.
